src/FlightMap/Widgets/QGCInstrumentWidget.qml

```diff
@@ -92,6 +92,37 @@ Rectangle {
                     onClicked:      _valuesWidget.showPicker()
                 }
             }
+
+            Image {
+                id:                 healthWarning
+                anchors.bottom:     outerCompass.bottom
+                anchors.left:       outerCompass.left
+                source:             "/qmlimages/Yield.svg"
+                mipmap:             true
+                visible:            _activeVehicle ? !_warningsViewed && _activeVehicle.unhealthySensors.length > 0 && _valuesWidget.currentPage() != 2 : false
+                opacity:            0.8
+                width:              outerCompass.width * 0.15
+                sourceSize.width:   width
+                fillMode:           Image.PreserveAspectFit
+
+                property bool _warningsViewed: false
+
+                MouseArea {
```

Comment:
Could you use a QGCMouseArea for this? This will cause it to automatically expand in size on touch systems. You set fillItem instead of doing an anchors.fill